src/test/java/levy/dictionary/EnglishDictionaryTest.java

```diff
@@ -0,0 +1,41 @@
+package levy.dictionary;
+import com.opencsv.exceptions.CsvValidationException;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class EnglishDictionaryTest {
+
+    @Test
+    public void testGetDefinition() {
+        try {
+            // given
+            EnglishDictionary dictionary = new EnglishDictionary();
+
+            // when
+            List<String> definitionsApple = dictionary.getDefinitions("apple");
+            List<String> definitionsBanana = dictionary.getDefinitions("banana");
+            List<String> definitionsCar = dictionary.getDefinition("car");
+            List<String> definitionsNonExistent = dictionary.getDefinition("nonexistent");
+
+            // then
+            assertEquals(1, definitionsApple.size());
+            assertEquals("a round fruit with red or green skin and white flesh", definitionsApple.get(0));
+
+            assertEquals(1, definitionsBanana.size());
+            assertEquals("a long curved fruit with a yellow skin", definitionsBanana.get(0));
+
+            assertEquals(1, definitionsCar.size());
+            assertEquals("a four-wheeled motor vehicle used for transportation", definitionsCar.get(0));
```

Comment:
You don't need all 3 of these.
You need to test words with multiple definitions.